components/views/hrm/employees/employee-list/employeelist/Employeelist.jsx

```diff
@@ -187,6 +209,27 @@ export default function Employeelist() {
 								icon_position={'right'}
 							/>
 						</div>
+							
+						<div className={'filter-dropdown'.classNames(EmployeelistCss)} style={{ minWidth: '113px' }}>
+							<DropDown
+								className={'padding-vertical-8 padding-horizontal-15'.classNames()}
+								placeholder={__('All Status')}
+								value={state.filters.employee_status}
+								onChange={(v) => onChange('employee_status', v)}
+								options={[
+									{ id: 0, label: __('All Status') },
```

Comment:
Not necessary to add anything with 0 to show initial state label. Simple pass this attribute `placeholder={__('All Status')}`